kyuubi-server/src/main/scala/org/apache/kyuubi/server/metadata/jdbc/JDBCMetadataStore.scala

```diff
@@ -432,6 +432,7 @@ class JDBCMetadataStore(conf: KyuubiConf) extends MetadataStore with Logging {
         val createTime = resultSet.getLong("create_time")
         val engineType = resultSet.getString("engine_type")
         val clusterManager = Option(resultSet.getString("cluster_manager"))
+        val engineOpenTime = resultSet.getLong("engine_open_time")
```

Comment:
```
    engine_open_time bigint, -- the engine open time
```

Reply:
Where is it assigned a value?

Reply:
let me check

Reply:
https://github.com/apache/kyuubi/blob/78e104d18b467dff5c97516c9dbe541eb8f7f276/kyuubi-server/src/main/scala/org/apache/kyuubi/operation/BatchJobSubmission.scala#L151